src/TauBounds.cpp

```diff
@@ -93,10 +95,11 @@ void computeLcas() {
 
 void initializeBounds() {
   for (int pop = 0; pop < dataSetup.popTree->numPops; pop++) {
+    tau_lbounds[pop] = 0.0;
     if (isLeafPop(pop)) {
-      tau_bounds[pop] = 0.0;
+      tau_ubounds[pop] = 0.0;
     } else {
-      tau_bounds[pop] = 100.0; // TODO - init with proper value (tau of father pop? MAX/INF value?)
+      tau_ubounds[pop] = 100.0; // TODO - init with proper value (tau of father pop? MAX/INF value?)
```

Comment:
There should be a symbolic constant for the maximum age in the system. I will check and get back to you.